<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowInvoicePage</class>
 <widget class="QWidget" name="ShowInvoicePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="CustomGraphicsView" name="graphicsView">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="buttonBack">
       <property name="text">
        <string>Späť</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QScrollBar" name="verticalScrollBar">
       <property name="maximum">
        <number>12</number>
       </property>
       <property name="pageStep">
        <number>25</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>Extensions/customgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
